### PR TITLE
[feat] Add an example on cross-lingual zero-shot transfer

### PR DESCRIPTION
**Pull request by tmynn**
Added a cross-lingual zero-shot transfer script example. Huggingface transformers and datasets are used.

### CODE REVIEW

File: examples/cross-lingual-transfer/main.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+                        param.requires_grad = False
+
+            # metric
+            self.metric = evaluate.load("accuracy")
+
+            # trackers
+            mlflow.set_tracking_uri(self.args.output_dir.joinpath("mlruns"))
+            os.environ["MLFLOW_EXPERIMENT_NAME"] = self.experiment_name
+
+            aim_callback = AimCallback(
```
Comment (alberttorosyan) [inline examples/cross-lingual-transfer/main.py:86]: Do we need to add `AimCallback`? Or the data should be synced with Aim using `aimlflow` CLI?

**Review by alberttorosyan — APPROVED** [review]
lgtm

File: examples/cross-lingual-transfer/main.py (hunk 1 above)
Comment (tmynn) [reply examples/cross-lingual-transfer/main.py:86]: Nice question @alberttorosyan.
In fact MLflow's callback is creating solid names (e.g. `eval_de_loss`). Thus making it hard(if not impossible) to explore the metrics on Aim UI, so using aimlflow is not the best option for MLflow's huggingface transformers callback.
However Aim's built-in callback for huggingface transformers has much better functionality and adds context to each metric (e.g. `loss` `context={"sub_dataset": "de", "subset": "eval"}`), making it super easy to work with these metrics on UI. I have consider these two cases in my use-case/blog post.

**Review by mihran113 — APPROVED** [review]
